atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -202,6 +203,52 @@ def run_task(self, fw_spec):
         return FWAction(stored_data=stored_data)
 
 
+@explicit_serialize
+class GetInterpolatedPOSCAR(FiretaskBase):
+    """
+    Grabs CONTCARS from two previous calculations to create interpolated
+    structure.
+
+    Required params:
+        start (str): name of fw for start of interpolation.
+        end (str): name of fw for end of interpolation.
+        this_image (int): which interpolation this is.
+        nimages (int) : number of interpolations.
+        autosort_tol (float): a distance tolerance in angstrom in which
```

Comment:
I don't understand the ``autosort_tol`` parameter

Reply:
(although I guess this is a pymatgen pass-through var, so maybe better if it's documented there)

Reply:
I've added a comment that it is a parameter for Structure.interpolate. I've also set a default and made it an optional parameter.